src/components/ConnectButton.tsx

```diff
@@ -0,0 +1,128 @@
+import { walletsAndConnectors } from "@/constants";
```

Comment:
we use a custom connect button instead of the standard one that ships with rainbowkit because we need to add our own styles.